src/glum/_glm.py

```diff
@@ -1968,10 +1971,10 @@ class GeneralizedLinearRegressor(GeneralizedLinearRegressorBase):
         specify it in parentheses (e.g., ``'tweedie (1.5)'``). The same applies
         for ``'negative.binomial'`` and theta parameter.
 
-    link : {'auto', 'identity', 'log', 'logit'} or Link, optional (default='auto')
-        The link function of the GLM, i.e. mapping from linear predictor
-        (``X * coef``) to expectation (``mu``). Option ``'auto'`` sets the link
-        depending on the chosen family as follows:
+    link : {'auto', 'identity', 'log', 'logit', 'cloglog'} or Link, optional
+        (default='auto') The link function of the GLM, i.e. mapping from linear
```

Comment:
```suggestion
    link : {'auto', 'identity', 'log', 'logit', 'cloglog'} or Link, optional (default='auto')
        The link function of the GLM, i.e. mapping from linear
```

Sphinx will be confused otherwise.